Makefile

```diff
@@ -80,10 +80,10 @@ release: dist ## package and upload a release
 	twine upload --skip-existing dist/alertlogic-sdk-python-*.* dist/*
 
 dist: clean ## builds source and wheel package
-	$(PYTHON) setup.py sdist
+	$(PYTHON) -m build
 
-install: clean ## install the package to the active Python's site-packages
-	$(PYTHON) setup.py install
+install: ## install the package to the active Python's site-packages
+	$(PIP) install pip install dist/alertlogic-sdk-python*.tar.gz
```

Comment:
To remove all dep warnings I think this should be  `$(PYTHON) -m pip install dist/alertlogic-sdk-python*.tar.gz` as pip doesn't like being called directly anymore

Reply:
Its more of a best practice I see as it does not throw a warning, will update it though to follow the best practices.

Reply:
Addressed via https://github.com/alertlogic/alertlogic-sdk-python/pull/114/commits/b60ce052216d260db89ac30baf384b4e369a59a7